README.md

```diff
@@ -264,5 +264,29 @@ For example:
   RtspTcpPort = "8554"
 ```
 
+
+### CameraStatus REST API Endpoint
```

Comment:
I think usually on Edgex end they refer to these as commands instead of REST apis. @lenny-intel your view on this?
```suggestion
### CameraStatus Command
```

Reply:
I like Command since it is using the Command Service